spec/system/onboardings/user_completes_onboarding_spec.rb

```diff
@@ -2,19 +2,21 @@
 
 RSpec.describe "Completing Onboarding", type: :system, js: true do
   let(:password) { Faker::Internet.password(min_length: 8) }
-  let(:user) { create(:user, password: password, password_confirmation: password, saw_onboarding: false) }
 
-  after do
-    sign_out user
```

Comment:
I realized this isn't actually necessary for this spec's purposes (and wasn't sure if the sign in/out process was causing the pundit error), so I went ahead and just removed it.